tests/integration/lxd/test_launcher.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import io
+import pathlib
+import subprocess
+
+import pytest
+
+from craft_providers import bases, lxd
+
+from . import conftest
+
+
+@pytest.fixture()
+def core20_instance():
+    with conftest.tmp_instance(
```

Comment:
why is tmp_instance in conftest?

Reply:
Because it's shared amongst the fixtures, I figured it was a reasonable spot to put it.  Would you prefer if I moved shared bits into its own module?